robots/pkg/test-report/test_report.go

```diff
@@ -267,43 +270,50 @@ func CreateJobNamePatternsToTestNameFilterRegexps(config *Config, client *http.C
 	jobNamePatternsToTestNameFilterRegexpsResult := map[*regexp.Regexp]*regexp.Regexp{}
 	for _, jobNamePatternToDontRunFileURL := range config.JobNamePatternsToDontRunFileURLs {
 		jobNamePattern := regexp.MustCompile(jobNamePatternToDontRunFileURL.JobNamePattern)
-		logger.Infof("fetching filter file %q", jobNamePatternToDontRunFileURL.DontRunFileURL)
-		response, err := client.Get(jobNamePatternToDontRunFileURL.DontRunFileURL)
+		dontRunFileURL := jobNamePatternToDontRunFileURL.DontRunFileURL
+		completeFilterRegex, err := createCompleteFilterRegexFromDontRunFileEntries(dontRunFileURL, client)
```

Comment:
is `Complete` redundant here? I found the function name quite long here and had to read it a couple of times. A shorter function name may help readablity.